src/Application.php

```diff
@@ -619,6 +639,20 @@ public function getFinalHandler(ResponseInterface $response = null)
     }
 
     /**
+     * Return the default delegate to use during `run()` when the stack is exhausted.
+     *
+     * Provided as a forwards compatibility measure with version 2, and proxies
+     * to getFinalHandler(); please note that it does not accept any arguments,
+     * nor pass them on to that method.
+     *
+     * @return callable|null
+     */
+    public function getDefaultDelegate()
```

Comment:
mhmm, sorry but it's not clear to me why would someone want to re-use it outside of `Application` context. Any example? :thinking: